content/sips/sip-120.md

```diff
@@ -222,7 +222,7 @@ Relevant only for atomic exchanges:
 
 Initially, this SIP proposes the following system configuration:
 
-- `SystemSettings.atomicMaxVolumePerBlock`: TBD
+- `SystemSettings.atomicMaxVolumePerBlock`: 500,000 sUSD
```

Comment:
@justinjmoses find the deployment params here.

We haven't discussed much on the max block volume but 500k was suggested as a safe amount to start with.